src/backend/metal/src/device.rs

```diff
@@ -1048,7 +1049,7 @@ impl hal::Device<Backend> for Device {
                 }
                 command::exec_blit(encoder, &soft::BlitCommand::CopyBuffer {
                     src: cpu_buffer.clone(),
-                    dst,
+                    dst: dst.to_owned(),
```

Comment:
Why is this needed now? Or is this simply preparing for future changes on the metal-rs side?

Reply:
Because we iterate over `&BufferRef` now. Previously this was done within `find`, so no performance/semantics change.